### DISCUSSION TITLE
Authentication with Keykloak

### DISCUSSION THREAD

**Discussion by DanielEdlich** [remsfal/remsfal-backend#226] [Ideas]
We should use Keycloak as an identity and access manger instead of Google. 
This would allow us to easily integrate other OAuth Providers besides Google and gives use the possibility to allow password-based login and user group management.

**Comment by Eyad-Wahdan** [comment]
This is an excellent proposal! Coincidentally, I had a similar idea a few weeks ago, along with some additional approaches and potential extensions. I have already discussed this with @astanik, and we are in active communication. I would be happy to keep you informed as developments unfold.
